chart/values.schema.json

```diff
@@ -521,7 +521,9 @@
                 "LocalKubernetesExecutor",
                 "CeleryExecutor",
                 "KubernetesExecutor",
-                "CeleryKubernetesExecutor"
+                "CeleryKubernetesExecutor",
+                "airflow.providers.amazon.aws.executors.batch.AwsBatchExecutor",
+                "airflow.providers.amazon.aws.executors.ecs.AwsEcsExecutor"
```

Comment:
@jedcunningham @LipuFei 

I don't know enough about Helm: will this import these modules from local sources (a repo checked out or something equivalent). Or does it depend on the changes from #39093 being published in the next provider release wave?

Reply:
All about the image - so the next provider being released and in the image the user configures.

Reply:
I think that is okay though, it's an experimental executor so expecting folks to be on a) the latest helm chart and b) the latest provider seems reasonable.

We don't know the provider version in the chart, so we can't really check at this level if things will work, we just have to deploy it. Should be a pretty obvious error message when things start crashlooping anyways :)

Reply:
Okay, I merged my other PR just now, I'll leave it up to you folks to use the shorter module paths or not going forward! :smiley: :+1:

Reply:
I kept the short names